processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/msi_1.asm (1 of 5)
processing file asmFiles/test.coherence1.asm (1 of 1)
--- memsim.hex	2022-04-12 17:53:36.634415738 -0400
+++ memcpu.hex	2022-04-12 17:53:36.554412758 -0400
@@ -1,4 +1,3 @@
-:0400000034080400BC
 :040001003C09DEAD2B
 :040002003529BEEFEF
 :04000300AD09000043
@@ -17,7 +16,7 @@
 :04008700AD090000BF
 :04008800FFFFFFFF78
 :04010000DEADBEEFC3
-:04010100BAD0BAD0E6
-:04010200BAD0BAD0E5
+:0401010089ABCDEF0A
+:040102000123456729
 :04010300CAB1FEED92
 :00000001FF
processing file asmFiles/test.coherence2.asm (1 of 1)
processing file asmFiles/test.coherence1.asm (1 of 1)
--- memsim.hex	2022-04-12 17:54:03.459415172 -0400
+++ memcpu.hex	2022-04-12 17:54:03.320409993 -0400
@@ -1,4 +1,3 @@
-:0400000034080400BC
 :040001003C09DEAD2B
 :040002003529BEEFEF
 :04000300AD09000043
@@ -17,7 +16,7 @@
 :04008700AD090000BF
 :04008800FFFFFFFF78
 :04010000DEADBEEFC3
-:04010100BAD0BAD0E6
-:04010200BAD0BAD0E5
+:0401010089ABCDEF0A
+:040102000123456729
 :04010300CAB1FEED92
 :00000001FF
processing file asmFiles/test.coherence1.asm (1 of 1)
processing file asmFiles/test.coherence2.asm (1 of 1)
processing file asmFiles/msi_1.asm (1 of 5)
--- memsim.hex	2022-04-12 17:59:07.824755615 -0400
+++ memcpu.hex	2022-04-12 17:59:07.723751851 -0400
@@ -8,5 +8,4 @@
 :040082008D090000E4
 :04008300AD0A0000C2
 :04008400FFFFFFFF7C
-:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_2.asm (2 of 5)
--- memsim.hex	2022-04-12 17:59:10.742864343 -0400
+++ memcpu.hex	2022-04-12 17:59:10.659861250 -0400
@@ -8,4 +8,5 @@
 :04008200AD090000C4
 :040083008D0A0000E2
 :04008400FFFFFFFF7C
+:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_3.asm (3 of 5)
--- memsim.hex	2022-04-12 17:59:13.671973481 -0400
+++ memcpu.hex	2022-04-12 17:59:13.588970389 -0400
@@ -9,4 +9,5 @@
 :040083008D0A0000E2
 :04008400AD0A0000C1
 :04008500FFFFFFFF7B
+:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_4.asm (4 of 5)
--- memsim.hex	2022-04-12 17:59:16.720087053 -0400
+++ memcpu.hex	2022-04-12 17:59:16.596082433 -0400
@@ -9,5 +9,4 @@
 :040082008D090000E4
 :04008300AD0A0000C2
 :04008500FFFFFFFF7B
-:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_5.asm (5 of 5)
--- memsim.hex	2022-04-12 17:59:19.758200253 -0400
+++ memcpu.hex	2022-04-12 17:59:19.646196080 -0400
@@ -9,5 +9,4 @@
 :04008300AD0A0000C2
 :04008400AD0A0000C1
 :04008500FFFFFFFF7B
-:04010000000000EE0D
 :00000001FF
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/msi_1.asm (1 of 5)
processing file asmFiles/msi_2.asm (2 of 5)
--- memsim.hex	2022-04-12 18:12:15.344098458 -0400
+++ memcpu.hex	2022-04-12 18:12:15.249094918 -0400
@@ -8,4 +8,5 @@
 :04008200AD090000C4
 :040083008D0A0000E2
 :04008400FFFFFFFF7C
+:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_3.asm (3 of 5)
--- memsim.hex	2022-04-12 18:12:18.373211322 -0400
+++ memcpu.hex	2022-04-12 18:12:18.270207485 -0400
@@ -9,4 +9,5 @@
 :040083008D0A0000E2
 :04008400AD0A0000C1
 :04008500FFFFFFFF7B
+:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_4.asm (4 of 5)
--- memsim.hex	2022-04-12 18:12:21.254318672 -0400
+++ memcpu.hex	2022-04-12 18:12:21.153314908 -0400
@@ -9,5 +9,4 @@
 :040082008D090000E4
 :04008300AD0A0000C2
 :04008500FFFFFFFF7B
-:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_5.asm (5 of 5)
processing file asmFiles/test.coherence2.asm (1 of 1)
processing file asmFiles/test.coherence1.asm (1 of 2)
--- memsim.hex	2022-04-12 18:50:59.911705987 -0400
+++ memcpu.hex	2022-04-12 18:50:59.761700398 -0400
@@ -1,4 +1,3 @@
-:0400000034080400BC
 :040001003C09DEAD2B
 :040002003529BEEFEF
 :04000300AD09000043
processing file asmFiles/test.coherence2.asm (2 of 2)
--- memsim.hex	2022-04-12 18:51:02.988820635 -0400
+++ memcpu.hex	2022-04-12 18:51:02.864816014 -0400
@@ -1,4 +1,3 @@
-:040000003408040CB0
 :040001008D09000065
 :04000200340A0010AC
 :040003000149480463
@@ -19,7 +18,7 @@
 :04008600AD2A00009F
 :04008700FFFFFFFF79
 :0401000000000001FA
-:04010200DEADBEEFC1
+:04010200DEAD00006E
 :040103000000DEAD6D
 :040105000000BEEF49
 :00000001FF
processing file asmFiles/test.coherence1.asm (1 of 1)
--- memsim.hex	2022-04-12 18:54:16.224020252 -0400
+++ memcpu.hex	2022-04-12 18:54:16.069014477 -0400
@@ -1,4 +1,3 @@
-:0400000034080400BC
 :040001003C09DEAD2B
 :040002003529BEEFEF
 :04000300AD09000043
processing file asmFiles/msi_1.asm (1 of 5)
--- memsim.hex	2022-04-12 18:57:27.225136631 -0400
+++ memcpu.hex	2022-04-12 18:57:27.139133427 -0400
@@ -1,4 +1,3 @@
-:0400000034080400BC
 :04000100340900EED0
 :04000200AD09000044
 :040006008D0A00005F
processing file asmFiles/msi_2.asm (2 of 5)
--- memsim.hex	2022-04-12 18:57:30.306251428 -0400
+++ memcpu.hex	2022-04-12 18:57:30.226248448 -0400
@@ -8,4 +8,5 @@
 :04008200AD090000C4
 :040083008D0A0000E2
 :04008400FFFFFFFF7C
+:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_3.asm (3 of 5)
--- memsim.hex	2022-04-12 18:57:33.470369318 -0400
+++ memcpu.hex	2022-04-12 18:57:33.336364325 -0400
@@ -9,4 +9,5 @@
 :040083008D0A0000E2
 :04008400AD0A0000C1
 :04008500FFFFFFFF7B
+:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_4.asm (4 of 5)
--- memsim.hex	2022-04-12 18:57:36.475481283 -0400
+++ memcpu.hex	2022-04-12 18:57:36.362477073 -0400
@@ -1,4 +1,3 @@
-:0400000034080400BC
 :04000100340900EED0
 :04000200AD09000044
 :040003008D0A000062
@@ -9,5 +8,4 @@
 :040082008D090000E4
 :04008300AD0A0000C2
 :04008500FFFFFFFF7B
-:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_5.asm (5 of 5)
--- memsim.hex	2022-04-12 18:57:39.691601111 -0400
+++ memcpu.hex	2022-04-12 18:57:39.592597422 -0400
@@ -1,4 +1,3 @@
-:0400000034080400BC
 :04000100340900EED0
 :04000200AD09000044
 :040003008D0A000062
processing file asmFiles/msi_5.asm (1 of 1)
--- memsim.hex	2022-04-12 19:11:32.604633619 -0400
+++ memcpu.hex	2022-04-12 19:11:32.486629222 -0400
@@ -1,4 +1,3 @@
-:0400000034080400BC
 :04000100340900EED0
 :04000200AD09000044
 :040003008D0A000062
processing file asmFiles/msi_5.asm (1 of 1)
--- memsim.hex	2022-04-12 19:12:24.400563399 -0400
+++ memcpu.hex	2022-04-12 19:12:24.269558518 -0400
@@ -1,4 +1,3 @@
-:0400000034080400BC
 :04000100340900EED0
 :04000200AD09000044
 :040003008D0A000062
processing file asmFiles/msi_5.asm (1 of 1)
processing file asmFiles/msi_1.asm (1 of 5)
processing file asmFiles/msi_2.asm (2 of 5)
--- memsim.hex	2022-04-12 19:26:41.347491014 -0400
+++ memcpu.hex	2022-04-12 19:26:41.240487027 -0400
@@ -8,4 +8,5 @@
 :04008200AD090000C4
 :040083008D0A0000E2
 :04008400FFFFFFFF7C
+:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_3.asm (3 of 5)
--- memsim.hex	2022-04-12 19:26:43.216560651 -0400
+++ memcpu.hex	2022-04-12 19:26:43.116556924 -0400
@@ -9,4 +9,5 @@
 :040083008D0A0000E2
 :04008400AD0A0000C1
 :04008500FFFFFFFF7B
+:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_4.asm (4 of 5)
--- memsim.hex	2022-04-12 19:26:45.053629095 -0400
+++ memcpu.hex	2022-04-12 19:26:44.945625071 -0400
@@ -9,5 +9,4 @@
 :040082008D090000E4
 :04008300AD0A0000C2
 :04008500FFFFFFFF7B
-:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_5.asm (5 of 5)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
--- memsim.hex	2022-04-12 19:27:01.143228553 -0400
+++ memcpu.hex	2022-04-12 19:27:01.097226839 -0400
@@ -19,7 +19,7 @@
 :04008600AD2A00009F
 :04008700FFFFFFFF79
 :0401000000000001FA
-:04010200DEADBEEFC1
+:04010200DEAD00006E
 :040103000000DEAD6D
 :040105000000BEEF49
 :00000001FF
processing file asmFiles/msi_1.asm (1 of 5)
processing file asmFiles/msi_2.asm (2 of 5)
--- memsim.hex	2022-04-12 19:27:34.269462752 -0400
+++ memcpu.hex	2022-04-12 19:27:34.175459250 -0400
@@ -8,4 +8,5 @@
 :04008200AD090000C4
 :040083008D0A0000E2
 :04008400FFFFFFFF7C
+:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_3.asm (3 of 5)
--- memsim.hex	2022-04-12 19:27:35.974526278 -0400
+++ memcpu.hex	2022-04-12 19:27:35.925524453 -0400
@@ -9,4 +9,5 @@
 :040083008D0A0000E2
 :04008400AD0A0000C1
 :04008500FFFFFFFF7B
+:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_4.asm (4 of 5)
--- memsim.hex	2022-04-12 19:27:37.812594760 -0400
+++ memcpu.hex	2022-04-12 19:27:37.690590214 -0400
@@ -9,5 +9,4 @@
 :040082008D090000E4
 :04008300AD0A0000C2
 :04008500FFFFFFFF7B
-:04010000000000EE0D
 :00000001FF
processing file asmFiles/msi_5.asm (5 of 5)
processing file asmFiles/msi_2.asm (1 of 1)
--- memsim.hex	2022-04-12 19:28:03.763561627 -0400
+++ memcpu.hex	2022-04-12 19:28:03.710559652 -0400
@@ -8,4 +8,5 @@
 :04008200AD090000C4
 :040083008D0A0000E2
 :04008400FFFFFFFF7C
+:04010000000000EE0D
 :00000001FF
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
--- memsim.hex	2022-04-12 19:29:16.253262409 -0400
+++ memcpu.hex	2022-04-12 19:29:16.150258571 -0400
@@ -19,7 +19,7 @@
 :04008600AD2A00009F
 :04008700FFFFFFFF79
 :0401000000000001FA
-:04010200DEADBEEFC1
+:04010200DEAD00006E
 :040103000000DEAD6D
 :040105000000BEEF49
 :00000001FF
